services/models.py

```diff
@@ -69,6 +69,13 @@ def __str__(self):
         return self.safe_translation_getter("title", super().__str__())
 
 
+class ServiceClientId(models.Model):
+    service = models.ForeignKey(
+        Service, on_delete=models.CASCADE, related_name="client_ids"
+    )
+    client_id = models.CharField(max_length=256, null=False, blank=False, unique=True)
```

Comment:
Wasn't the max length supposed to be 255?

Reply:
I already had this code before the 255 values came up and it was so close so I didn't change it. I don't think it needs to be that strict here. The main point is that every possible client id fits into this field.

Reply:
255 would be consistent with fields in many other models in the project, so someone could think that 256 is somehow special. But doesn't really matter.

Reply:
That seems to be the case. It's a bit weird. Usually these kind of limits are powers of two (I don't know why, perhaps programmers just think in binary). That's also visible in this code base. You can find 32, 64, 128, 1024... and then... **255** 🤯 I don't have an explanation to this discrepancy.